### PR TITLE
Add a scraper for pyvista

### PR DESCRIPTION
**Pull request by Louis-Pujol**
Hi ! I create a new PR as I forked again the project after some troubles with conflicting versions of the code.

In this PR I propose an implementation of a scraper for the [pyvista](https://docs.pyvista.org/version/stable/) library.

The scraper is a slight modification of the [pyvista's sphinx-gallery scraper](https://github.com/pyvista/pyvista/blob/main/pyvista/plotting/utilities/sphinx_gallery.py). I integrated it following the mayavi scraper already implemented.

It allows to show static or animated 3d scenes created with pyvista. I added an example to the examples gallery. I moved the definition of image_scrapers from mkdocs.yml to gallery_conf.py. I found this clearer as pyvista is configured in gallery_conf.py.

I also added pyvista as a dependency in MKDOCS_GALLERY_EXAMPLES_REQS in noxfile.py.

### CODE REVIEW

**Comment by Louis-Pujol** [comment]
Seems to work, should I add a line to the 0.8.0 section of the changelog file or create a section 0.8.1 ?

**Comment by smarie** [comment]
Thanks @Louis-Pujol  ! Yes, please create a new changelog section (0.9.0) with this item. This is significant enough :)

**Comment by Louis-Pujol** [comment]
Ok I do that. In the meantine, I has also work on the following :
- Solve #54
- Add a scraper for [vedo](https://vedo.embl.es/) (another VTK-based 3D visualization library)

Would you prefer me to create new PRs for these changes, or to merge them directly here ?

**Comment by smarie** [comment]
Thanks ! No, please Create distinct PRs, that will be easier to review

File: docs/changelog.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 0.9.0 - Pyvista
+
+- Pyvista can now be used in gallery examples as in `sphinx-gallery`. PR [#91](hhttps://github.com/smarie/mkdocs-gallery/pull/91) by [Louis-Pujol](https://github.com/Louis-Pujol)
```
Comment (smarie) [inline docs/changelog.md:5]:
```suggestion
- Pyvista can now be used in gallery examples as in `sphinx-gallery`. PR [#91](https://github.com/smarie/mkdocs-gallery/pull/91) by [Louis-Pujol](https://github.com/Louis-Pujol)
```

**Comment by smarie** [comment]
0.9.0 released with your contribution : https://smarie.github.io/mkdocs-gallery/generated/gallery/plot_11_pyvista/
Thanks !

**Pull request merged**